packages/playground/cli/src/run-cli.ts

```diff
@@ -153,6 +153,20 @@ export async function parseOptionsAndRunCLI() {
 				type: 'boolean',
 				default: false,
 			})
+			.option('wp-config-default-constants', {
+				describe:
+					'Configure default constant values to use in "wp-config.php", in case the constants are missing. Encoded as JSON string.',
+				type: 'string',
+				coerce: (value: string) => {
+					try {
+						return JSON.parse(value);
```

Comment:
Is this CLI option a JSON because there's no easy way to distinguish between numbers, strings, and booleans with a text-based CLI flag? It's a bit awkward but I guess it makes sense.